games-action/minetest/metadata.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<herd>games</herd>
```

Comment:
Is the games herd being a pity illusion of itself not reason enough to drop it? Games herd did nothing for this ebuild and i am the sole maintainer of it and i as a maintainer decide that i don't need nor want the games herd.